test/JDBC/input/nullableDatetime.sql

```diff
@@ -0,0 +1,57 @@
+-- [BABEL-2769] Nullable DATETIME column does not store NULL
+
+CREATE DATABASE db_babel_2769;
```

Comment:
no need to use a separate database instead create objects with filename_xx as name, for example nullableDatetime_smalldatetime and so on

Reply:
creating all the tables in master db instead

Reply:
using master db instead